tests/integration/collection_description/updates/replace/materialized_test.go

```diff
@@ -0,0 +1,121 @@
+// Copyright 2024 Democratized Data Foundation
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package replace
+
+import (
+	"testing"
+
+	"github.com/sourcenetwork/immutable"
+
+	"github.com/sourcenetwork/defradb/client"
+	testUtils "github.com/sourcenetwork/defradb/tests/integration"
+)
+
+func TestColDescrUpdateReplaceIsMaterialized_GivenFalseAndCollection_Errors(t *testing.T) {
+	test := testUtils.TestCase{
+		Actions: []any{
+			testUtils.SchemaUpdate{
+				Schema: `
+					type User {
+						name: String
+					}
+				`,
+			},
+			testUtils.PatchCollection{
+				Patch: `
+					[
+						{ "op": "replace", "path": "/1/IsMaterialized", "value": false }
+					]
+				`,
+				ExpectedError: "non-materialized collections (only views) are not supported. Collection: User",
```

Comment:
suggestion: This reads a little weird. I would suggest to move the parenthesis at the end `non-materialized collections are not supported. (only views)`